File: src/main/java/net/flyingfishflash/loremlist/api/MaintenanceApiService.java
package net.flyingfishflash.loremlist.api;

import net.flyingfishflash.loremlist.api.data.response.ApiServiceResponse;
import net.flyingfishflash.loremlist.api.data.response.DomainPurgedResponse;

public interface MaintenanceApiService {
  ApiServiceResponse<DomainPurgedResponse> purge();
}

File: src/main/kotlin/net/flyingfishflash/loremlist/api/MaintenanceApiService.kt
package net.flyingfishflash.loremlist.api

import net.flyingfishflash.loremlist.api.data.response.ApiServiceResponse
import net.flyingfishflash.loremlist.api.data.response.DomainPurgedResponse

interface MaintenanceApiService {
  fun purge(): ApiServiceResponse<DomainPurgedResponse>
}
